Fix test failure on armhf and ppc64el:
https://gitlab.freedesktop.org/spice/spice-server/issues/31

Taken from upstream:
https://gitlab.freedesktop.org/spice/spice/commit/19f9f454e0777d851f26d14df0c7984267c57015

diff --git a/server/tests/test-qxl-parsing.c b/server/tests/test-qxl-parsing.c
index 60ca8f88c62441e02577ced21e4f60a08ad4171a..234bdabc9ce619d0799b5136f1d72357b0b2f490 100644
--- a/server/tests/test-qxl-parsing.c
+++ b/server/tests/test-qxl-parsing.c
@@ -96,7 +96,7 @@ static void test_memslot_invalid_slot_id(void)
     RedMemSlotInfo mem_info;
     init_meminfo(&mem_info);
 
-    memslot_get_virt(&mem_info, 1 << mem_info.memslot_id_shift, 16, 0);
+    memslot_get_virt(&mem_info, UINT64_C(1) << mem_info.memslot_id_shift, 16, 0);
 }
 
 static void test_memslot_invalid_addresses(void)
